tests/min_max_test.py

```diff
@@ -78,6 +78,19 @@ def test_quantized_bits():
       assert expected[1] == q.max()
 
 
+@pytest.mark.parametrize('alpha', [None, 2.0])
+@pytest.mark.parametrize('symmetric,keep_negative', 
+                         [(True, True), (False, True), (False, False)])
+@pytest.mark.parametrize('bits', [1, 8])
+def test_quantized_linear(bits, symmetric, keep_negative, alpha):
+
+    q = quantized_linear(bits=bits, 
+                         symmetric=symmetric, 
+                         keep_negative=keep_negative, 
+                         alpha=alpha)
+    assert q(-1000) == q.min()
+    assert q(1000)== q.max()
```

Comment:
Consider add more test cases, such as quantized values within min/max range and those that are right on the boundary.

Reply:
Done